about.html

```diff
@@ -0,0 +1,214 @@
+                <div class="para_about">
+                    The International Summit brings together the community of experts, academics, and activisits who comprisw
+                    the creative Commons affiliate network in a different country every two year. This year, we hope to
+                    expand our inviation list including organisations and individuals. 
+                </div>
+                <div class="about-contact">
+                    Please contact us per Email for any further questions about Coding Minds Summit  2015
+                    <br>
+                    <a class="summit_mail" href="#">codingmindsint2015@cminternational.org</a>
+                </div>
+
+            </div>
+
+        </div>
+
+        <div class="logo_section">
+            <h2 class="logo_head">
+                Coding Minds International Summit 2015 Logo
+            </h2>
+            <hr class="about_them">
+           
+            <p class="logo_des">
+                The logo of Coding Minds International was decide through the logo competion from 8.June to 7.july
+            </p>
+
+            <p class="logo_desk">
+                The logo of Coding Minds International was decide through the logo competion from 8.June to 7.july.
+                After online public voting and Selection Communities judging process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimised the prize-winning-logo for this year's summit as following:
+            </p>
+
+            <div class="logo_container">
+
+                <div class="log2_cont">
+
+                    <div class="log2">
+                        <img class="log2_img" src="images/logo.png" alt="conference logo">
+                        <div class="log2_title">
+                            <div class="log2_title1">CM</div>
+                            <div class="log2_title2">Summit 2015</div>
+                        </div>
+                    </div>
+
+                    <div class="logo_date">14th-17th Oct | K'la, Uganda</div>
+
+                </div>          
+                
+            </div>
+        </div>
+
+        <hr class="logo_divider">
+        <section class="past_summit">
+            <h3 class="summit_head">
+                See the past CM International Summits
+            </h3>
+            <hr class="about_them">
+            <p class="summit_para">
+                Take a look at the las two Kampala Programmers Forums which took place in Buenos Aire and Warsaw.
+            </p>
+
+            <div class="summit_img">
+                <div class="summit_img_one">
+                    <div class="year"><strong>2013</strong></div> <br>Kampala Programmers Forum 2013 in Buenos Aires
+                </div>
+    
+    
+                <div class="summit_img_two">
+                    <div class="year"><strong>2011</strong></div> <br>Kampala Programmers Forum 2013 in Warsaw
+                </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please `add` required color fiter to your 2 pictures like a [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 


![image](https://user-images.githubusercontent.com/63915024/219433185-188e83d1-0d97-464b-9ce4-db7c2074359a.png)